chain/indexer/src/block.rs

```diff
@@ -236,6 +236,7 @@ where
                                         Ok(Some(event_type)) => {
                                             // Pair the event type with the TX hash here
                                             let significant_event = SignificantChainEvent { tx_hash, event_type };
+                                            info!("indexer got {significant_event}");
```

Comment:
Maybe a debug would be better?

Reply:
I think there should be at least some output on Indexer activity visible in `info` level.
Currently there's only `Processed block #...`

We can move it back once we migrate to `tracing`